db/1.9/modules/ROOT/pages/database-connector-troubleshooting.adoc

```diff
@@ -42,19 +78,58 @@ The following example shows how to use pooling profiles with the *Stored procedu
 	</flow>
 ----
 
-== Snowflake JDBC Driver Throws an Exception for a Stored Procedure
+
+== Troubleshoot Snowflake JDBC Driver that Throws an Exception for a Stored Procedure
 
 Some drivers have their own specifications when calling stored procedures. Running the application when you set the callable statement with curly brackets (`{` and `}`), for example, `{call doubleMyInt(:myInt)}`, causes the Snowflake JDBC driver to throw the following exception:
 
 `net.snowflake.client.jdbc.SnowflakeSQLException: Unsupported feature 'call'`
 
 
-=== Solution
+To resolve this issue:
 
 For Snowflake JDBC driver, set the callable statement without the curly brackets `{` and `}`, for example:
 
 `call doubleMyInt(:myInt)`
 
+
+== Understand Common Throws
+
+Here is a list of common throws messages and how to interpret them.
+
+* DB:BAD_SQL_SYNTAX
+
+ The provided SQL query has a bad syntax.
+
+* DB:CANNOT_LOAD_DRIVER
+
+ Database Connector can not load the JDBC driver.
+
+* DB:CANNOT_REACH
+
+ Database Connector can not establish a connection with the RDBMS.
+
+* DB:CONNECTIVITY
+
+ A connectivity issue with the database that occurs on badly configured connections, no database response while waiting results or fetching rows, lost of database connection, and similars.
+
+* DB:INVALID_DATABASE
+
+ The connection to the RDBMS is established but the database doesn't exist.
+
+* DB:INVALID_CREDENTIALS
+
+ The database rejects the provided credentials.
+
+* DB:QUERY_EXECUTION
+
+ There is an error executing the query.
```

Comment:
This just repeats the error message; is there any value to add here, maybe some possible causes?